patroni/ctl.py

```diff
@@ -1193,9 +1193,8 @@ def reinit(cluster_name: str, group: Optional[int], member_names: List[str], for
                 wait_on_members.remove(member)
 
 
-def _do_failover_or_switchover(action: str, cluster_name: str, group: Optional[int],
-                               switchover_leader: Optional[str], candidate: Optional[str],
-                               force: bool, scheduled: Optional[str] = None) -> None:
+def _do_failover_or_switchover(action: str, cluster_name: str, group: Optional[int], candidate: Optional[str],
+                               force: bool, leader: Optional[str] = None, scheduled: Optional[str] = None) -> None:
```

Comment:
Maybe we shouldn't rename switchover_leader -> leader?
After all, it is used exclusively for a switchover.

Reply:
well, scheduled is also only for switchover